dynawo/sources/Modeler/DataInterface/PowSyblIIDM/test/TestDataInterface.cpp

```diff
@@ -1054,15 +1129,19 @@ TEST(DataInterfaceIIDMTest, testHvdcLineVscConvertersIIDM) {
   hvdc->setValue(HvdcLineInterfaceIIDM::VAR_STATE1, OPEN);
   hvdc->setValue(HvdcLineInterfaceIIDM::VAR_STATE2, OPEN);
   data->exportStateVariablesNoReadFromModel();
-  ASSERT_DOUBLE_EQUALS_DYNAWO(vsc1->getVscIIDM().getTerminal().getP(), -200.);
-  ASSERT_DOUBLE_EQUALS_DYNAWO(vsc1->getVscIIDM().getTerminal().getQ(), -400.);
-  ASSERT_DOUBLE_EQUALS_DYNAWO(vsc2->getVscIIDM().getTerminal().getP(), -600.);
-  ASSERT_DOUBLE_EQUALS_DYNAWO(vsc2->getVscIIDM().getTerminal().getQ(), -800.);
+  ASSERT_DOUBLE_EQUALS_DYNAWO(vsc1->getVscIIDM().getTerminal().getP(), 150.);
```

Comment:
Why this change?

Reply:
Just before, vsc1 P attribute is set to 150. Since the hvdc isn't connected, the P attribute remains at 150.

Before this task, disconnected HVDC used to have their attributes changed but now it's not the case anymore.